From e296f5c824321d5c12be3121d8159fe6a248c3fc Mon Sep 17 00:00:00 2001
From: Colin Walters <walters@verbum.org>
Date: Sun, 8 Jan 2012 12:11:11 -0500
Subject: [PATCH] build: Make --disable-xml-docs build work again

We can't build the .html files without xmlto, so don't add them
to a dist_ variable.

https://bugs.freedesktop.org/show_bug.cgi?id=55426
---
 doc/Makefile.am |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index b265987..db81b5c 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -59,9 +59,6 @@ STATIC_HTML = \
 
 dist_html_DATA += $(STATIC_HTML)
 
-# we distribute these in the tarball so users don't necessarily need xmlto
-dist_html_DATA += $(XMLTO_OUTPUT)
-
 XMLTO_OUTPUT=					\
 	dbus-faq.html				\
 	dbus-specification.html			\
@@ -69,6 +66,10 @@ XMLTO_OUTPUT=					\
 	dbus-tutorial.html
 
 if DBUS_XML_DOCS_ENABLED
+
+# we distribute these in the tarball so users don't necessarily need xmlto
+dist_html_DATA += $(XMLTO_OUTPUT)
+
 dbus-specification.html: dbus-specification.xml
 	$(XMLTO) html-nochunks $<
 
-- 
1.7.10.4

